docs/docs-beta/docs/etl-pipeline-tutorial/refactor-your-project.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Refactor your project
+description: Refactor your completed project into a structure that is more organized and scalable. 
+last_update:
+  author: Alex Noonan
+sidebar_position: 80
+---
+
+Many engineers generally leave something alone once its working as expected. But the first time you do something is rarely the best implementation of a use case and all projects benefit from incremental improvements.
+
+## Splitting up project structure
+
+Right now the project is contained within one definitions file. This has gotten kinda unwieldy and if we were to add more to the project it would only get more disorganized. So we're going to create separate files for all the different Dagster core concepts: 
```

Comment:
I would rewrite this a bit: "Currently, your project is contained in one definitions file. However, this file has gotten fairly complex, and adding to it would only increase its complexity. To fix that, we will create separate files for each core Dagster concept"